scheduler/src/cook/mesos/api.clj

```diff
@@ -1385,7 +1407,9 @@
              :handler (read-jobs-handler conn fid task-constraints gpu-enabled? is-authorized-fn)}
        :post {:summary "Schedules one or more jobs."
               :parameters {:body-params RawSchedulerRequest}
-              :responses {200 {:schema [JobResponse]}}
+              :responses {201 {:description "The jobs were successfully scheduled."}
+                          400 {:description "One or more of the jobs were incorrectly specified."}
+                          409 {:description "One or more of the jobs UUIDs were already in use."}}
```

Comment:
@shamsimam This string is used in the Swagger documentation for the endpoint, but is never included in a response generated by the endpoint.  So, it's static - can't mention the number of jobs in a particular request here.